terraform/modules/matrix-service/lambdas/driver_lambda.tf

```diff
@@ -120,8 +120,8 @@ resource "aws_lambda_function" "matrix_service_driver_lambda" {
     variables = {
         DEPLOYMENT_STAGE = "${var.deployment_stage}"
         DYNAMO_REQUEST_TABLE_NAME="dcp-matrix-service-request-table-${var.deployment_stage}"
-        MATRIX_QUERY_BUCKET = "dcp-matrix-service-queries-${var.deployment_stage}"
-        MATRIX_RESULTS_BUCKET = "dcp-matrix-service-results-${var.deployment_stage}"
+        MATRIX_QUERY_BUCKET = "dcp-matrix-service-query-results-${var.deployment_stage}"
+        MATRIX_QUERY_RESULTS_BUCKET = "dcp-matrix-service-results-${var.deployment_stage}"
```

Comment:
I think you wanted to update the value of this to the new bucket

Reply:
Ah good catch! I made a mess of these variables... will clean up